R/CalculateChange.R

```diff
@@ -0,0 +1,143 @@
+#' Calculate Change
+#'
+#' Calculate the change from the previous snapshot date.
```

Comment:
Let's add some more text here about expected behavior when there is only a single snapshot or a value is missing in a previous snapshot.

Reply:
(Looks like NA is being returned, which is fine)